src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -2285,7 +2285,12 @@ public final String getDidlString(RendererConfiguration mediaRenderer) {
 			media.isAudio() &&
 			StringUtils.isNotBlank(firstAudioTrack.getSongname())
 		) {
-			title = firstAudioTrack.getSongname();
+			title = "";
+			if (mediaRenderer != null && mediaRenderer.isPrependTrackNumbers() && firstAudioTrack.getTrack() > 0) {
+				// zero pad for proper numeric sorting on all devices
+				title += String.format("%03d: ", firstAudioTrack.getTrack());
```

Comment:
I think some devices don't accept `:` in filenames (like my TV) so it would be more safe to do like `"%03d - "` (it would be a current bug that we can fix here)

Reply:
That seems reasonable to me.